mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -944,13 +955,140 @@ class BinaryLogisticRegressionSummary private[classification] (
 private class LogisticAggregator(
     private val numFeatures: Int,
     numClasses: Int,
-    fitIntercept: Boolean) extends Serializable {
+    fitIntercept: Boolean,
+    multinomial: Boolean,
+    standardize: Boolean) extends Serializable {
 
   private var weightSum = 0.0
   private var lossSum = 0.0
 
+  private val totalCoefficientLength = {
+    val cols = if (fitIntercept) numFeatures + 1 else numFeatures
+    val rows = if (multinomial) numClasses else math.max(1, numClasses - 1)
```

Comment:
The current behavior allows `LogisticAggregator` to be called with `multinomial = true` and `numClasses = 2`. This will essentially be the same as binary LR, but will return two sets of coefficients instead of one. This is akin to calling R glmnet with multinomial family and numClasses = 2.